modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/client_http_invoker.ftl

```diff
@@ -263,7 +263,7 @@ public class HttpInvoker {
 		fileName = fileName.replaceAll("\"", "");
 		fileName = fileName.replaceAll("\n", "");
 		fileName = fileName.replaceAll("\r", "");
-	
+
```

Comment:
@jeffwu0724 This change does not belong to this commit, please merge it into your previous commit.